unreleased.md

```diff
@@ -3,3 +3,9 @@
 **Added:**
 - react-native-bpk-component-pagination-dots:
   - Introducing the React Native pagination dots component.
+
+- bpk-component-icon:
+  - bpk-mixins:
```

Comment:
These need to be added to the changelog too given that we directly expose the icon api through these components:
- `react-native-bpk-component-button`
- `react-native-bpk-component-button-link`
- `react-native-bpk-component-navigation-bar`